packages/manager/modules/exchange/src/exchange.service.js

```diff
@@ -1755,7 +1755,21 @@ export default class Exchange {
           exchangeService: name,
         },
       },
-    ).then(this.getSuccessDataOrReject);
+    );
+  }
+
+  updateExchangeServer(organization, name, data) {
+    return this.services.OvhHttp.put(
```

Comment:
Cn't it be replaced by a `ovh-api-services` call ?

Reply:
Oh I forgot it after `git subtree pull`

Reply:
Need to release a new version (https://github.com/ovh-ux/ovh-api-services/pull/273)